authors/apps/bookmark/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from .views import CreateBookmark, ListALlBookmarks, DeleteBookmark
+
+
+app_name = "bookmarks"
+urlpatterns = [
+    path('article/<slug>/create/bookmark/', CreateBookmark.as_view(), name='create-bookmark'),
+    path('article/<slug>/delete/bookmark/', DeleteBookmark.as_view(), name='delete-bookmark'),
```

Comment:
Since we are building a rest API, we should have one endpoint create and delete bookmarks